### DISCUSSION TITLE
RP2040 possibility

### DISCUSSION THREAD

**Discussion by BangDroid** [Drumber/RemoteLight#19] [General]
How hard would it be to port this to the Raspi Pico? I'm  pretty much an MCU newb, but I imagine it's not as simple as compiling for the Pico in Arduino IDE.

**Comment by Drumber** [comment]
I haven't really looked into the Raspberry Pi Pico that much. However, it should be possible to port the Glediator protocol to the Pico, because there is already a Neopixel library for it and it can receive date from the serial port.
The Pico must therefore be connected to the PC via USB and RemoteLight could then send the LED data via the serial port.
But I don't have time to create a sketch for it at the moment. I would get back to you when I have a sketch ready.
